modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -18,18 +19,128 @@ As an Operator author, your Operator must meet additional requirements to run pr
 // TODO: Include more info w/ better steps on how to do this:
 //* You must understand the {product-title} proxy configuration.
 
-For the CSV requirements, you can make the following changes as the Operator author.
+For the CSV requirements, you can make the following changes as the Operator author. The following procedure uses the Memcached Operator created in the Operator SDK tutorials for Go, Ansible, and Helm.
 
 .Prerequisites
 
 * An Operator project with a CSV.
 
 .Procedure
 
-. Use SHA references to related images in two places in the CSV for your Operator:
+. Add the images and tags used by the Operator by setting an environment variable, such as `RELATED_IMAGE_MEMCACHED`, in the `config/manager/manager.yaml` file:
```

Comment:
Specifically, I would call out that these are for additional images used by the operator. For example with Memcached operator, the operator image itself will be automagically resolved to a digest, but additional images, such as the actual Memcached image, will need to be specified here so they can be resolved.